src/actions/playground.ts

```diff
@@ -0,0 +1,27 @@
+import { Action, ActionCreator } from 'redux'
+
+/**
+ * The `type` attribute for an `Action` which updates the `IPlaygroundState`
+ * `editorValue`
+ */
+export const UPDATE_EDITOR_VALUE: string = 'UPDATE_EDITOR_VALUE'
```

Comment:
`string` is redundant

Reply:
When in doubt, you can check with your  editor /IDE, if upon hovering to the variable you see a type, that means the type can be inferred.

Of course sometimes you want to explicitly declare it anyway, for documentation purposes